test/helper/config_tarantool.lua

```diff
@@ -0,0 +1,188 @@
+local fio = require('fio')
+local t = require('luatest')
+local fun = require('fun')
+local yaml = require('yaml')
+
+local server = require('test.helper.server')
+
+local config = {}
+
+local roles = {
+    'roles.sharded-queue-router',
+    'roles.sharded-queue-storage',
+}
+config.root = fio.dirname(fio.abspath(package.search('init')))
+config.datadir = fio.pathjoin(config.root, 'dev')
+config.unitdir = fio.pathjoin(config.datadir, 'unit')
+config.configpath = fio.pathjoin(config.root, 'test', 'entrypoint', 'config.yml')
+config.devconfigpath = fio.pathjoin(config.root, 'dev', 'config.yml')
+
+function config.start_example_replicaset()
+    local opts = {
+        config_file = config.devconfigpath,
+        chdir = config.datadir,
+    }
+    local servers = {
+        "queue-router",
+        "queue-router-1",
+        "queue-storage-1-0",
+        "queue-storage-1-1",
+        "queue-storage-2-0",
+        "queue-storage-2-1",
+    }
+    config.servers = {}
+    for _, server_name in ipairs(servers) do
+        local server_opts = fun.chain(opts, {alias = server_name}):tomap()
+        config.servers[server_name] = server:new(server_opts)
+    end
+
+    for _, srv in pairs(config.servers) do
+        srv:start({wait_until_ready = false})
+    end
+
+    for _, srv in pairs(config.servers) do
+        srv:wait_until_ready()
+    end
+    config.servers['queue-router']:eval("require('vshard').router.bootstrap()")
```

Comment:
Let's add `if_not_bootstrapped = true` just in case

Reply:
Thank you, updated.